src/Documentation/2.0/Migration1.5.md

```diff
@@ -8,6 +8,9 @@ title: Migration from Orleans 1.5 to 2.0
 The bulk of the Orleans APIs stayed unchanged in 2.0 or implementation of those APIs were left in legacy classes for backward compatibility. At the same time, the newly introduced APIs provide some new capabilities or better ways of accomplishing those tasks. There are also more subtle differences when it comes to .NET SDK tooling and Visual Studio support that helps to be aware of. This document provides guidance for migrating application code from to Orleans 2.0.
 
 ## Visual Studio and Tooling requirements
+Orleans 2.0.0 is built on top of .Net Standard 2.0. Because of that, you need to upgrade development tools to ensure yourself a pleasant developing experience. We recommend to use Visual Studio 15 or above to develop Orleans 2.0.0 applications. Based on our experience, version 15.4 and above works best. 
```

Comment:
nit: I've seen that the .NET team are particular about the capitalization of .NET

Reply:
"Visual Studio 15" might be confusing to some: better to call it Visual Studio 2017 or *maybe* a full version like 15.5.2